src/pycram/worlds/multiverse_extras/exceptions.py

```diff
@@ -0,0 +1,6 @@
+from pycram.datastructures.enums import JointType
```

Comment:
Change to relative import and move this to pycram/failures.py

Reply:
done